### PR TITLE
Add back galaxy watch 4

### PR DESCRIPTION
**Pull request by YayunHuang**
Ref: MUP-133

https://github.com/user-attachments/assets/fc159f74-36cc-4b22-bc0b-e5faea277fe7

### CODE REVIEW

File: src/scripts/device_info.json [hunk 1]
```diff
@@ -6752,6 +6752,123 @@
       ],
       "is_mockup_image_at_front": true,
       "available_perspectives": ["Portrait", "Landscape"]
+    },
+    {
+      "credits": "<p><a href=\"http://facebook.design/devices\" target=\"blank\">Facebook - Design Resources</a></p>",
```
Comment (pkong-ds) [inline src/scripts/device_info.json]: This should be skymaker, can search `skymaker` in same file for copying

File: src/scripts/device_info.json [hunk 2]
```diff
@@ -6752,6 +6752,123 @@
...
+            [405, 400]
+          ],
+          "name": "landscape"
+        }
+      ],
+      "is_mockup_image_at_front": true,
+      "available_perspectives": ["Portrait", "Landscape"]
+    },
+    {
+      "credits": "<p><a href=\"http://facebook.design/devices\" target=\"blank\">Facebook - Design Resources</a></p>",
```
Comment (pkong-ds) [inline src/scripts/device_info.json]: same

File: src/scripts/device_info.json [hunk 3]
```diff
@@ -6752,6 +6752,123 @@
...
```
Comment (pkong-ds) [inline src/scripts/device_info.json]: same

**Review by pkong-ds — CHANGES_REQUESTED** [review]

**Comment by YayunHuang** [comment]
@pkong-ds updated, thanks

**Review by pkong-ds — APPROVED** [review]

**Pull request merged**